src/SDKs/SqlManagement/Management.Sql/Microsoft.Azure.Management.Sql.csproj

```diff
@@ -7,7 +7,7 @@
 		<PackageId>Microsoft.Azure.Management.Sql</PackageId>
 		<Description>Azure SQL Management SDK library</Description>
 		<AssemblyName>Microsoft.Azure.Management.Sql</AssemblyName>
-		<Version>1.24.0-preview</Version>
+		<Version>1.25.0-preview</Version>
```

Comment:
Please remove release notes from 1.24 and add new release notes.

Reply:
Updated release notes